docker-build/Dockerfile

```diff
@@ -1,16 +1,10 @@
-FROM alpine:3.6
-
-LABEL description="Docker image for caddy+forwardproxy plugin."
-LABEL maintainer="SergeyFrolov@colorado.edu"
-
-RUN apk add --no-cache ca-certificates bash curl
-
-RUN curl --fail https://getcaddy.com | bash -s http.forwardproxy
-
-COPY gen_caddyfile_and_start.sh /bin/
-
-VOLUME /root/.caddy
-
-EXPOSE 80 443 2015
-
-ENTRYPOINT /bin/gen_caddyfile_and_start.sh
+FROM golang:1.14.15 AS builder
+RUN git clone https://github.com/caddyserver/forwardproxy.git
+WORKDIR /go/forwardproxy/cmd/caddy 
+RUN go build caddy.go
+
+FROM ubuntu:20.04
+COPY --from=builder /go/forwardproxy/cmd/caddy/caddy /usr/bin/caddy
+EXPOSE 80 443
+ENTRYPOINT ["/usr/bin/caddy"] 
+CMD ["-conf", "/etc/caddy/Caddyfile", "--log", "/dev/stdout", "--agree=true"]
```

Comment:
where did /etc/caddy/Caddyfile come from?

Reply:
This case is good for Kubernetes - there you can put a config file as Config Map. I can describe a solution for Kubernetes, or put a default config in docker. In any case, it seems to me that it would be good to start the application with a configuration file.

Reply:
How will it be more convenient?

Reply:
Currently existing `gen_caddyfile_and_start.sh` script (if it's still functional, haven't touched in a while) generates the config automatically. That seems convenient